<launch>
    <arg name="map_server" default="1" />
    <node pkg="changemap" name="sub_changemap" type="sub_changemap.py" />
    <group if="$(arg map_server)">
        <include file="$(find mushr_pf)/launch/real.launch" />
        <include file="$(find mushr_rhc_ros)/launch/real/cp_real.launch" />
    </group>

</launch>
